hospital_management_system/hospital_app/models.py

```diff
@@ -1,43 +1,59 @@
 import uuid
 from django.db import models
 from django.contrib.auth.models import User
-
+from rest_framework.authtoken.models import Token as DefaultToken
+from django.core.validators import RegexValidator
+from datetime import datetime
 
 class Hospital(models.Model):
-    models.UUIDField(primary_key = True,default=uuid.uuid4,unique=True,editable=False)
-    name = models.CharField(max_length=200,default='')
-    address = models.TextField(max_length=1000,default='')
-    Staffs_num = models.IntegerField()
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, unique=True, editable=False)
+    name = models.CharField(max_length=200, default='')
+    address = models.TextField(max_length=1000, default='')
+    staffs_num = models.IntegerField()
     num_of_beds = models.IntegerField(unique=True)
 
-
     def __str__(self):
-        return str(self.name)
-    
+        return self.name
+
 class Patient(models.Model):
-    models.UUIDField(primary_key = True,default=uuid.uuid4,unique=True,editable=False)
-    patient_name = models.CharField(max_length=200,default='')
-    contact_number = models.CharField(max_length=200,default='',null=False,unique=True)
-    department =  models.CharField(max_length=200,default='')
-    disease = models.CharField(max_length=200,default='')
-    patient_address = models.CharField(max_length=200,default='')
-    
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, unique=True, editable=False)
+    patient_name = models.CharField(max_length=200, default='')
+    contact_number = models.CharField(
+        max_length=10,
+        validators=[
+            RegexValidator(
+                regex=r'^\d{10}$',
+                message='Phone number must be exactly 10 digits',
+                code='invalid_phone_number'
+            ),
+        ],
+        
+    )
+    department = models.CharField(max_length=200, default='')
+    disease = models.CharField(max_length=200, default='')
+    patient_address = models.CharField(max_length=200, default='')
+    hospital = models.ForeignKey(Hospital, related_name='patients', on_delete=models.CASCADE)
```

Comment:
on POST success for empty details
handle it

Reply:
fixed